compiler/passes/normalize.cpp

```diff
@@ -978,6 +944,43 @@ static TypeSymbol* resolveTypeAlias(SymExpr* se)
   return NULL;
 }
 
+//
+// These RiSpec helpers implement reduce intents of the form
```

Comment:
You know, I have no idea what RiSpec means. Maybe describe that in the comment and/or choose a better function name?